2023-02-01 14:50:55,638                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$5(RedisConnectionPool.java:45)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool$$Lambda$12/575593575.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
2023-02-01 14:50:55,642                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$5(RedisConnectionPool.java:45)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool$$Lambda$12/575593575.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
2023-02-01 14:50:55,643                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$5(RedisConnectionPool.java:45)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool$$Lambda$12/575593575.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
2023-02-01 14:50:55,643                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$5(RedisConnectionPool.java:45)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool$$Lambda$12/575593575.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
2023-02-01 14:50:55,643                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$5(RedisConnectionPool.java:45)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool$$Lambda$12/575593575.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
2023-02-01 15:01:24,089                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$93/1060032248.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 15:01:24,090                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$93/1060032248.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 15:01:24,090                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$93/1060032248.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 15:01:24,090                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$93/1060032248.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 15:01:24,091                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$93/1060032248.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 16:01:26,130                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$34/2045458791.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 16:01:26,131                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$34/2045458791.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-01 16:01:26,131                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$34/2045458791.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-02 13:17:46,527                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$396/541256278.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-02 13:17:46,530                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$396/541256278.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-02 13:17:46,532                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$396/541256278.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-02 13:17:46,534                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$396/541256278.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-02 13:17:46,536                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$396/541256278.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-02 13:17:46,537                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$397/1313172554.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-02 13:17:46,538                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$397/1313172554.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-02 14:31:50,401                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$400/687597334.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-02 14:31:50,402                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$400/687597334.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-02 14:31:50,402                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$400/687597334.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-02 14:31:50,403                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$400/687597334.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-02 14:31:50,403                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketException: Software caused connection abort: socket write error
	at redis.clients.jedis.Connection.disconnect(Connection.java:221)
	at redis.clients.jedis.Connection.close(Connection.java:210)
	at redis.clients.jedis.Jedis.close(Jedis.java:290)
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$400/687597334.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
Caused by: java.net.SocketException: Software caused connection abort: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:109)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at redis.clients.jedis.util.RedisOutputStream.flushBuffer(RedisOutputStream.java:57)
	at redis.clients.jedis.util.RedisOutputStream.flush(RedisOutputStream.java:138)
	at redis.clients.jedis.Connection.disconnect(Connection.java:217)
	... 6 common frames omitted
2023-02-03 13:29:20,564                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$431/2110459285.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-03 13:29:20,567                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$431/2110459285.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-03 13:29:20,569                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$431/2110459285.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-03 13:29:20,571                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$431/2110459285.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-03 13:29:20,572                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.sql.SQLRecoverableException: IO Error: An existing connection was forcibly closed by the remote host
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1992)
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:2748)
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at vn.vnpay.receiver.thread.ShutdownThread$$Lambda$431/2110459285.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at oracle.net.nt.TimeoutSocketChannel.read(TimeoutSocketChannel.java:415)
	at oracle.net.ns.NSProtocolNIO.doSocketRead(NSProtocolNIO.java:1118)
	at oracle.net.ns.NIOPacket.readHeader(NIOPacket.java:266)
	at oracle.net.ns.NIOPacket.readPacketFromSocketChannel(NIOPacket.java:198)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:140)
	at oracle.net.ns.NIOPacket.readFromSocketChannel(NIOPacket.java:113)
	at oracle.net.ns.NIONSDataChannel.readDataFromSocketChannel(NIONSDataChannel.java:97)
	at oracle.jdbc.driver.T4CMAREngineNIO.prepareForUnmarshall(T4CMAREngineNIO.java:833)
	at oracle.jdbc.driver.T4CMAREngineNIO.unmarshalUB1(T4CMAREngineNIO.java:486)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:621)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:298)
	at oracle.jdbc.driver.T4C7Ocommoncall.doOLOGOFF(T4C7Ocommoncall.java:73)
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:1982)
	... 5 common frames omitted
2023-02-06 09:19:43,892                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionPool.lambda$getInstancePool$0(KafkaConsumerConnectionPool.java:42)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionPool$$Lambda$14/1511785794.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
2023-02-06 22:47:28,159                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
2023-02-06 22:47:28,160                     close:53   - [] - connection is closed: {0} - e.OracleConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.oracle.OracleConnectionCell.close(OracleConnectionCell.java:51)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:24)
2023-02-08 18:44:59,482                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$0(RedisConnectionPool.java:45)
	at java.lang.Thread.run(Thread.java:750)
2023-02-08 18:44:59,483                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$0(RedisConnectionPool.java:45)
	at java.lang.Thread.run(Thread.java:750)
2023-02-08 18:44:59,483                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$0(RedisConnectionPool.java:45)
	at java.lang.Thread.run(Thread.java:750)
2023-02-08 18:44:59,485                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$0(RedisConnectionPool.java:45)
	at java.lang.Thread.run(Thread.java:750)
2023-02-08 18:44:59,486                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at vn.vnpay.receiver.connect.redis.RedisConnectionPool.lambda$getInstancePool$0(RedisConnectionPool.java:45)
	at java.lang.Thread.run(Thread.java:750)
2023-02-08 18:46:32,644                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-08 18:46:32,644                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-08 18:46:32,645                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-08 18:46:32,645                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-08 18:46:32,645                     close:45   - [] - connection is closed: {0} - is.RedisConnectionCell 
java.lang.NullPointerException: null
	at vn.vnpay.receiver.connect.redis.RedisConnectionCell.close(RedisConnectionCell.java:43)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:25)
2023-02-08 21:46:45,717                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,718                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,718                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,718                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,719                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,719                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,719                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,719                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,720                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:46:45,720                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,301                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,303                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,306                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,306                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,306                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,307                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,307                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,307                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,308                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:53:14,308                     close:52   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:49)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:58:17,704                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 21:58:17,713                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:00:39,317                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:00:39,333                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:02:15,529                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:02:15,564                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:03:32,296                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:03:32,337                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:04:29,784                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:04:29,796                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:05:54,890                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:05:54,897                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:06:30,594                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:06:30,598                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:20:33,726                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:20:33,731                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:24:47,123                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:24:47,123                     close:55   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:52)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:26:53,455                     close:57   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:54)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:27:28,535                     close:57   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:54)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:30:24,619                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:30:24,620                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:30:24,620                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:30:24,620                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:30:24,620                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:33:20,781                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:33:20,781                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:33:20,782                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:33:20,782                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:33:20,782                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,857                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,858                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,858                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:37:41,859                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,144                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,146                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,148                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,154                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,155                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,155                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,155                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,155                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,155                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:47:10,156                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,635                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,635                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,636                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,636                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,636                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,636                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,636                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,637                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,637                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:50:33,637                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,038                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,043                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,043                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,043                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,043                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,044                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,046                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,046                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,047                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-08 22:51:33,047                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,422                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,422                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 00:23:09,423                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,472                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,474                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,474                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,474                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,474                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,474                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,475                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,475                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,475                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
2023-02-09 23:24:10,475                     close:59   - [] - connection is closed: {0} - ConsumerConnectionCell 
java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
	at org.apache.kafka.clients.consumer.KafkaConsumer.acquire(KafkaConsumer.java:2445)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2328)
	at org.apache.kafka.clients.consumer.KafkaConsumer.close(KafkaConsumer.java:2283)
	at vn.vnpay.receiver.connect.kafka.KafkaConsumerConnectionCell.close(KafkaConsumerConnectionCell.java:56)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at vn.vnpay.receiver.thread.ShutdownThread.run(ShutdownThread.java:28)
